src/runtime_src/core/pcie/tools/xbmgmt/cmd_version.cpp

```diff
@@ -15,9 +15,9 @@
  */

```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`